packages/move/tests/test_end_to_end.move

```diff
@@ -62,6 +62,13 @@ module deployment_addr::test_end_to_end {
         let user1_balance = primary_fungible_store::balance(user1_addr, fa_metadata_object);
         assert!(user1_balance == 0, user1_balance);
 
+        // FIX: for more thorough testing you'll want to add other assertions such as VestingStream, etc state being initialized correctly
+        // and that the relevant fungible store increased it's holding appropriately.
+
+        // FIX: instead of using timestamp::update_global_time_for_test_secs to update the time
+        // you could simply use timestamp::fast_forward_seconds to fast forward a specified number of seconds
+        // making the tests for easily understandable
```

Comment:
I started out with the fast_forward_seconds, but for me the update_global_time_for_test_secs was more readable. Especially since there are 3 time jumps. With my approach, I can interpret the time by just reading a single block of the test. But again, this is just a matter of preference. In an existing codebase, I would ofcourse follow the standard that has been set with other tests to ensure its uniform.